geomstats/datasets/prepare_emg_data.py

```diff
@@ -17,6 +17,8 @@
 class TimeSeriesCovariance:
     """Class for generating a list of covariance matrices from time series.
 
+    Prepare a TimeSeriesCovariance Object from time series in dictionary.
+
     Parameters
     ----------
```

Comment:
Sorry I didn't see this earlier, but could we replace `time_vec` by `time`? `time_vec` seems less natural there, and you also use `time` in the emg.csv data. It can seem like a small detail, but using `time` everywhere will be more intuitive and consistent, thus improving the user experience. Thanks!

Reply:
Second NIT on code design/organization: could we rename `n_elecs` into `n_timeseries`? 

Since your code works for any multi-timeseries, we will later add it to the module that transforms data into data on manifolds:
- e.g. put graphs on hyperbolic spaces,
- e.g. put multi-timeseries on spd manifolds, through their covariance matrices.
Thus, if we start writing general code, that's awesome and will save us time down the road!